### DISCUSSION TITLE
Problems exporting from MoBi to PK-Sim

### DISCUSSION THREAD

**Discussion by PatriciaRega** [Open-Systems-Pharmacology/Forum#1744] [Questions & Problems]
Hello everyone,

We've encountered a problem with the 11.2 version of PK-Sim. Specifically, we're experiencing difficulties when attempting to export MoBi projects to PK-Sim again for the purpose of conducting population simulations. 

Here is a screenshot of the warning message we get:

![PKSIM issue](https://github.com/Open-Systems-Pharmacology/Forum/assets/162401288/17b13750-bb58-40a7-a4d3-5bde64ca8160)

Then it isn't possible to open the project.

Has anyone else experienced similar issues or found a workaround? Any insights or suggestions would be greatly appreciated.

Thank you in advance for your assistance.

Best regards,
Patricia

**Comment by Ricardo-DO** [comment]
Hi Patricia,

Probably is not the same as your issue, but once happened to me that I loaded some observed data without specifying the molecule in PKSim, or I think I loaded directly to MoBi. The observed data was showing as ng/mL, but because it was missing the link to the molecule (molecular weight), it couldn't convert the units to uM.

The solution I found it was to always load the observed data in PKSim using Add Observed Data for -> "Molecule" instead of using Add Observed Data.

Hope it helps,
Ricardo

**Comment by Yuri05** [comment]
Do you have observed data sets with undefined molecular weight in your MoBi project?
If so: try to assign the correct MW before sending a simulation to PK-Sim
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/6939c6b8-ac4b-4832-b88a-21a56846e0ca)
